### ISSUE TITLE
add arg to support delete of named kv root

### ISSUE THREAD

**Issue by bitsofinfo** [bitsofinfo/files-to-consul-kv#1]
add arg to support delete of named kv root

**Comment by josephlim75** [comment]
agreed with @bitsofinfo .  Like the simplicity of this tool.  The only constraints seems like it doesn't take into consideration of deltas such as update or delete.  If you can support deltas update/delete from git, that would be great.  By supporting deltas, you don't stress out consul because you only update what needs to be updated from git

**Comment by bitsofinfo** [comment]
@josephlim75 please feel free to submit a PR!
